translation.rst

```diff
@@ -670,8 +655,9 @@ priority message files.
 The filename of the translation files is also important: each message file
 must be named according to the following path: ``domain.locale.loader``:
 
-* **domain**: An optional way to organize messages into groups (e.g. ``admin``,
-  ``navigation`` or the default ``messages``) - see :ref:`using-message-domains`;
+* **domain**: An optional way to organize messages into groups. Unless
+  parts of the application are explicitly separated from eachother, it is
```

Comment:
I don't know it it's correct, but `eachother` is more commonly spelled as `each other`

Reply:
Thanks, turns out `eachother` is wrongly spelling so I've changed it to `each other`.